src/test/lib/logical_query_plan/lqp_translator_test.cpp

```diff
@@ -1163,9 +1163,19 @@ TEST_F(LQPTranslatorTest, TranslatePrunableSubqueries) {
   const auto pqp = LQPTranslator{}.translate_node(lqp);
   const auto& get_table = std::dynamic_pointer_cast<const GetTable>(pqp->left_input());
   ASSERT_TRUE(get_table);
-  const auto& prunable_subquery_scans = get_table->prunable_subquery_scans();
+  const auto& prunable_subquery_scans = get_table->prunable_subquery_predicates();
   ASSERT_EQ(prunable_subquery_scans.size(), 1);
   EXPECT_EQ(prunable_subquery_scans.front(), pqp);
 }
 
+TEST_F(LQPTranslatorTest, WindowNode) {
```

Comment:
Minor: use a more descriptive test name that states which assertion you're testing here?

Reply:
(Discussed offline) only tests that we fail since there's currently no operator
https://github.com/hyrise/hyrise/blob/dd39b51058659aed8affea2e93f95e8ab0453665/src/lib/logical_query_plan/lqp_translator.cpp#L507-L510